pkgs/applications/networking/bee/bee-clef.nix

```diff
@@ -0,0 +1,71 @@
+{ version ? "release", stdenv, lib, substituteAll
+, fetchFromGitHub, go-ethereum }:
+
+let
+
+  versionSpec = {
+    release = rec {
+      name = "bee-clef";
+      version = "0.4.4";
+      rev = "refs/tags/v${version}";
+      sha256 = "18fab0nxxsglrdykhb02m6vd7pgd6z4pqgfp5nmpxjcz7s8rfnml";
+    };
+  }.${version};
+
+in
+
+stdenv.mkDerivation {
+  pname = versionSpec.name;
+  version = versionSpec.version;
+
+  src = fetchFromGitHub {
+    owner = "ethersphere";
+    repo = "bee-clef";
+    rev = versionSpec.rev;
+    sha256 = versionSpec.sha256;
+  };
+
+  buildInputs = [ go-ethereum ];
+
+  patches = [
+    (substituteAll {
+      src = ./bee-clef.patch;
+      clef = "${go-ethereum}/bin/clef";
+      inherit outputDir;
+
+      # this fails, but i also need to somehow inject paths that are referring to files in $out 
+      #??? inherit out;
+    })
+  ];
+
+  buildPhase = " "; # we don't want the Makefile to be run
+
+  # this intentionally reproduces the same directory structure as it
+  # is in the upstream repo, but only copies the files that are
+  # needed.
+  installPhase = ''
+    mkdir -p $out/lib/systemd/system
```

Comment:
Why create an empty directory? Maybe explain the reasons more in the comment above.

Reply:
oh, that's a bug i just introduced with one of the hasty edits.

the `bee-clef.service` fie needs to go there for the systemd machinery to pick it up, but i have accidentally deleted that `cp`.